src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/ParenPadCheck.java

```diff
@@ -101,18 +101,78 @@
  * &lt;module name=&quot;ParenPad&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Foo {
+ *
+ *   public Test(int y ) {  // violation, space before right parenthesis
+ *   }
+ *
+ *   public void foo() {  // OK
+ *     bar( 1);  // violation, space after left parenthesis
+ *   }
+ *
+ *   public void bar(int k ) {  // violation, space before right parenthesis
+ *     while (k &gt; 0) {  // OK
+ *     }
+ *
+ *     Test obj = new Test(k);  // OK
+ *   }
+ *
+ *   public void fun() {  // OK
+ *     switch(y ) {  // violation, space before right parenthesis
+ *       default:
+ *         break;
+ *     }
+ *   }
+ *
+ * }
+ * </pre>
+ * <p>
  * To configure the check to require spaces for the
  * parentheses of constructor, method, and super constructor calls:
  * </p>
  * <pre>
  * &lt;module name=&quot;ParenPad&quot;&gt;
- *   &lt;property name=&quot;tokens&quot; value=&quot;CTOR_CALL, METHOD_CALL,
+ *   &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_FOR, LITERAL_CATCH,
  *     SUPER_CTOR_CALL&quot;/&gt;
  *   &lt;property name=&quot;option&quot; value=&quot;space&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
- * The following cases not checked:
+ * Example:
+ * </p>
+ * <pre>
+ * class Foo {
+ *
+ *   int x;
+ *
+ *   public Foo(int n) {
+ *   }
+ *
+ *   public void fun() {
+ *     try {
+ *       System.out.println(x);
+ *     } catch( Exception e) {  // violation, no space before right parenthesis
+ *     }
+ *
+ *     for ( int i = 0; i &lt; x; i++ ) {  // OK
+ *     }
+ *   }
+ *
+ * }
+ *
+ * class Bar extends Foo {
+ *
+ *   public Bar() {
+ *     super(1 );  // violation, no space after left parenthesis
```

Comment:
` no space after left parenthesis`
???

Reply:
This example, and the example above display the ```space``` padding... That is why I wrote ```no space``` in both of them

Reply:
it is not clear (at least for me) what causes violation. 
Please add examples for correct code to show difference